program_intent_engagement/apps/api/v1/views.py

```diff
@@ -112,12 +113,16 @@ def grab_best_intents(cls, user_intents):
         ):
             intents = list(intents)
 
-            # sort by certainty, effective_timestamp
+            # sort by effective_timestamp descending, then certainty ascending
```

Comment:
Updated so that it is chaining multiple sorts. Needed to keep the lambda for the sorting of certainty since it is using the certainty order value to sort which is a separate dict. Lemme know what you think @ashultz0